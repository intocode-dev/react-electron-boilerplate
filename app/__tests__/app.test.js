import React from 'react';
import App from '../app';

test('render app', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
