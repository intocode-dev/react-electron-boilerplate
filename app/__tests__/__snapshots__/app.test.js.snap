// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<div>
  React Electron Boilerplate by intocode Co., Ltd.
</div>
`;
